clock_exercise/clockmanager.h

```diff
@@ -0,0 +1,41 @@
+#ifndef CLOCKMANAGER_H
+#define CLOCKMANAGER_H
+
+#include <QObject>
+#include <QTimer>
+#include <memory>
+
+class ClockManager : public QObject
+{
+    Q_OBJECT
+public:
+    explicit ClockManager(QObject *parent = nullptr);
+
+signals:
+    void dateTextChanged();
+    void minutesClockHandAngleChanged();
+    void hoursClockHandsAngleChanged();
+
+public slots:
+
+private:
+    void refreshClockHandsAngles();
+    void refreshDateText();
+
+    std::unique_ptr<QTimer> mTimerForRefresh;
+
+    // ↓↓↓ properties
```

Comment:
I commenti che indicano dove si trovano le property togliamoli, come dicevo nell'altro commento mi aspetto che le property siano in cima.